src/scheduler/MpiWorld.cpp

```diff
@@ -1227,21 +1213,6 @@ long MpiWorld::getLocalQueueSize(int sendRank, int recvRank)
     return queue->size();
 }
 
-void MpiWorld::checkRankOnThisHost(int rank)
```

Comment:
Even if we decide to keep the `if` statements, we can delete this method and keep the check in `getLocalQueue` through `getHostForRank`.